api/lib/schema.js

```diff
@@ -271,6 +273,7 @@ const typeDefs = gql`
     voteOption: String
     lockedBalance: Float
     timeLock: String
+    numberOfSeconds: Int
```

Comment:
why does this need to be a variable?

Reply:
it is needed to create a valid second transaction in Polkadot